kolibri/plugins/setup_wizard/assets/src/views/importFacility/SelectFacilityForm.vue

```diff
@@ -75,24 +67,30 @@
       header() {
         return this.getCommonSyncString('selectFacilityTitle');
       },
+      importDeviceId() {
+        return this.wizardService._state.context.importDeviceId;
+      },
       selectedFacility() {
         return this.facilities.find(f => f.id === this.selectedFacilityId);
       },
     },
+    mounted() {
+      console.log('ImportAuthentication', this.wizardService.state);
```

Comment:
and here. now I'm wondering if there is a particular reason these are being logged? if so - sorry :D

Reply:
Nope I just didn't do a log audit before making the PR :sweat: